filter_test.go

```diff
@@ -44,8 +45,10 @@ func TestLoadFilteredPolicy(t *testing.T) {
 	testHasPolicy(t, e, []string{"admin", "domain2", "data2", "read"}, true)
 
 	if err := e.LoadFilteredPolicy(&fileadapter.Filter{
-		P: []string{"", "domain1"},
-		G: []string{"", "", "domain1"},
+		PG: map[string][][]string{
```

Comment:
A way to keep compatibility:

```go
	if err := e.LoadFilteredPolicy(&fileadapter.Filter{
		P: []string{"", "domain1"},
		G: []string{"", "", "domain1"},
		G2: []string{"", "", "domain1"},
		G3: []string{"", "", "domain1"},
	}); err != nil {
		t.Errorf("unexpected error in LoadFilteredPolicy: %v", err)
	}
```

I think we can hard-code it to G4 or G5. It's enough to use.